python/cuml/test/dask/test_comms.py

```diff
@@ -143,13 +143,15 @@ def test_send_recv(n_trials, ucx_cluster):
 
 @pytest.mark.ucx
 @pytest.mark.parametrize("n_trials", [5])
-def test_recv_any_rank(n_trials, ucx_cluster):
+@pytest.mark.skip(reason="Should be enabled once this feature is fixed: "
+                         "https://github.com/rapidsai/cuml/issues/1975")
+def test_recv_any_rank(n_trials, cluster):
 
-    client = Client(ucx_cluster)
+    client = Client(cluster)
 
     try:
 
-        cb = CommsContext(comms_p2p=True)
+        cb = CommsContext(comms_p2p=True, verbose=True)
```

Comment:
I think this might be from my PR, but let’s change it back to False here in case tthis is merged first.